client/extensions/zoninator/state/locks/selectors.js

```diff
@@ -0,0 +1,76 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import { get } from 'lodash';
+
+const getLocksState = state => get( state, 'extensions.zoninator.locks', {} );
+
+/**
+ * Returns the lock expiration date for the specified site and zone IDs.
+ *
+ * @param  {Object} state  Global state tree
+ * @param  {Number} siteId Site ID
+ * @param  {Number} zoneId Zone ID
+ * @return {Number}        Lock expiration date
+ */
+export const getLock = ( state, siteId, zoneId ) =>
+	get( getLocksState( state ), [ 'items', siteId, zoneId ], 0 );
```

Comment:
Ah-hah! 😄 The fallback value here (and in selectors below) might indicate could initial values for the reducers above (in which case you might be able to drop them here).